activepieces/activepieces
# feat(brilliant-directories): create User action

## What does this PR do?
Adds a new piece called Brilliant Directories and an action to 'create user'. Created on behalf of a client who wishes to make this piece public.

To test the piece, you'll need an account on Brilliant Directories, I can provide this.

Otherwise to test:
1. Add Brilliant Directories action 'create user' to the flow
2. Make a connection, using the API key + Brilliant Directories Site URL
3. Fill out the forms as required
4. The meta form should be filled out as JSON, like so:
`{
"username": "shaypunter",
}`

ANNOUNCEMENT=true
